docs/admin/access.rst

```diff
@@ -510,3 +499,10 @@ however re-create them if you delete or rename them.
     Never remove the predefined Weblate groups and users, as this can lead to
     unexpected problems. If you have no use for them, you can removing all their
     privileges instead.
+
+Locking down Weblate
+++++++++++++++++++++
+
+If you experience problems with malice on your Weblate installation,
```

Comment:
```suggestion
If you want ultimate control of who signs up for the platform, or experience problems with malice on your Weblate installation,
```
Unless this is true?

Reply:
I've looked back into origin of the passage: it was introduced around weblate 1.3; the original intention (as far as I can tell) was to document a way to create a most private weblate installation, see #131.

That's how it ended up in `Common setups`: it's a way to setup the weblate as private as possible, i.e. if you only want to add new users manually and forbid anybody who's unregistered from accessing any page. `Locking down` is kinda misleading here...

See suggestion bellow.

Reply:
"Ultimate control" is a paradox of insufficiency. It sounds too good.

Reply:
`paradox of insufficiency` ??